extension/src/popup/Router.tsx

```diff
@@ -214,6 +216,9 @@ export const Router = () => {
         <PublicKeyRoute path={ROUTES.account}>
           <Account />
         </PublicKeyRoute>
+        <PublicKeyRoute path={ROUTES.accountHistory}>
+          <AccountHistory />
+        </PublicKeyRoute>
```

Comment:
I added new routes for Account History and Settings to make the bottom nav highlight the active tab